omahelsinki/settings.py

```diff
@@ -184,3 +196,9 @@
             secret.close()
         except IOError:
             Exception('Please create a %s file with random characters to generate your secret key!' % secret_file)
+
+REST_FRAMEWORK = {
```

Comment:
We're not using the REST Framework in this project, so this setting is unnecessary.